src/test/java/com/uwetrottmann/trakt/v2/services/CheckinTest.java

```diff
@@ -2,15 +2,7 @@
 
 import com.uwetrottmann.trakt.v2.BaseTestCase;
 import com.uwetrottmann.trakt.v2.TestData;
-import com.uwetrottmann.trakt.v2.entities.EpisodeCheckin;
-import com.uwetrottmann.trakt.v2.entities.EpisodeCheckinResponse;
-import com.uwetrottmann.trakt.v2.entities.EpisodeIds;
-import com.uwetrottmann.trakt.v2.entities.MovieCheckin;
-import com.uwetrottmann.trakt.v2.entities.MovieCheckinResponse;
-import com.uwetrottmann.trakt.v2.entities.MovieIds;
-import com.uwetrottmann.trakt.v2.entities.ShareSettings;
-import com.uwetrottmann.trakt.v2.entities.SyncEpisode;
-import com.uwetrottmann.trakt.v2.entities.SyncMovie;
+import com.uwetrottmann.trakt.v2.entities.*;
```

Comment:
One more thing: no star imports please :) In most cases your IDE is configured to do this automatically (it's good practice to turn it off).